components/org.apache.stratos.cloud.controller/src/main/java/org/apache/stratos/cloud/controller/statistics/publisher/DASMemberStatusPublisher.java

```diff
@@ -38,14 +38,27 @@
 public class DASMemberStatusPublisher extends ThriftStatisticsPublisher implements MemberStatusPublisher {
 
     private static final Log log = LogFactory.getLog(DASMemberStatusPublisher.class);
+    private static volatile DASMemberStatusPublisher dasMemberStatusPublisher;
     private static final String DATA_STREAM_NAME = "member_lifecycle";
     private static final String VERSION = "1.0.0";
     private static final String DAS_THRIFT_CLIENT_NAME = "das";
+    private static final int STATS_PUBLISHER_THREAD_POOL_SIZE = 10;
```

Comment:
STATS_PUBLISHER_THREAD_POOL_SIZE same thing we have declared in DASMemberInformationPublisher. Better we can move this constant in to common place. May be we can move to the CloudControllerConstants.